plugins/modules/system/open_iscsi.py

```diff
@@ -368,6 +400,22 @@ def main():
             result['changed'] |= True
             result['automatic_changed'] = True
 
+    if automatic_portal is not None:
+        isauto = target_isauto(module, target, portal, port)
+        if (automatic_portal and isauto) or (not automatic_portal and not isauto):
+            result['changed'] |= False
+            result['automatic_portal_changed'] = False
+        elif not check:
+            if automatic_portal:
+                target_setauto(module, target, portal, port)
+            else:
+                target_setmanual(module, target, portal, port)
+            result['changed'] |= True
+            result['automatic_portal_changed'] = True
+        else:
+            result['changed'] |= True
+            result['automatic_portal_changed'] = True
```

Comment:
If you add a new return value, you need to docment it (see `RETURN`).

Reply:
Sorry I dont understand where should I document it.

Reply:
Also now that `automatic_portal` has been removed, so this change var does not resemble change of `auto_portal_statup`, isnt it?

Reply:
Hmm, looks like this module doesn't document any return values at all. Return value documentation looks like this: https://github.com/ansible-collections/community.general/blob/f9973c55214c90f34578c3fde2ee2bef3a0ac325/plugins/modules/system/dconf.py#L69

Since there currently is no documentation, I don't mind if you don't document this one as well. Whatever you prefer.

> Also now that automatic_portal has been removed, so this change var does not resemble change of auto_portal_statup, isnt it?

This return variable represents whatever you want it to represent. It probably makes sense to name it more closely to the option.